.github/workflows/main.yml

```diff
@@ -30,6 +30,7 @@ jobs:
     - name: Install standard dependencies
       run: |
         pip install -r requirements.txt
+        pip install pysatCDF
```

Comment:
I wonder if installing with the no-binary option would help with the bits where it fails?